modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/containers/setup-forms/InviteTeamMembersForm/index.js

```diff
@@ -31,6 +31,12 @@ import TeamMemberInputs from './TeamMemberInputs';
 const MAXIMUM_INVITES_COUNT = 10;
 const INITIAL_INVITES_COUNT = 1;
 
+const DEFAULT_ERROR = {
+	message: i18n.translate('an-unexpected-error-occurred'),
+	title: i18n.translate('error'),
+	type: 'warning',
+};
```

Comment:
Since it's a warning. I think it's best a different alert ahaha.
Like something wrong may have happened

Reply:
Whaaat?!!?! I just looked at it, I forgot to amend it hahaha